src/main/scala/org/hammerlab/guacamole/reads/Read.scala

```diff
@@ -93,7 +81,7 @@ object Read extends Logging {
    * @param record
    * @return
    */
-  def fromSAMRecord(record: SAMRecord): Read = {
```

Comment:
Interesting point. In general I don't like giving constructors names that are redundant with the types that they expect; the arguments will commonly have descriptive names already, like the sole caller of the ADAM version here where we now have:

```
adamRecords.map(Read(_))
```

instead of

```
adamRecords.map(Read.fromADAMRecord(_))
```

which is preferable to me, so I think I'll leave these changed.